crane-cloud/frontend
# Implement Admin Dashboard Overview

# Description

This PR showcases a provisional implementation of the admin dashboard overview which summarizes the user, database, and cluster stats and metrics for purposes of monitoring.

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

https://trello.com/c/gjOayz5W

## How Can This Been Tested?

Pull and run this branch locally and visit the clusters route on the admin page to notice the added features.


## Checklist:


# Screenshots

![image](https://user-images.githubusercontent.com/63339234/137034348-3fb1ea96-a10f-46f7-a84c-cc12d22d4f58.png)
